custom_components/frigate/const.py

```diff
@@ -33,6 +33,9 @@
 ATTR_FAVORITE = "favorite"
 ATTR_MQTT = "mqtt"
 
+# Frigate Attribute Labels
+ATTRIBUTE_LABELS = ["amazon", "face", "fedex", "license_plate", "ups"]
```

Comment:
It will likely change but the changes will be known because the label will have to be added to the list in frigate+ and then it wouldn't be possible to have it in the model until some time after once examples have been submitted. The only one I know right now that is likely to be added is `USPS`

Reply:
OK --  sounds like it will change, but it's more a release action, than something we need to convert to be dynamic somehow.

Reply:
Right, I agree, I don't think it will become a burden requiring frequent patches or anything like that